recipe/conda_build_config.yaml

```diff
@@ -3,3 +3,23 @@
 # available on 10.13
 MACOSX_SDK_VERSION:        # [osx and x86_64]
   - "10.13"                # [osx and x86_64]
+c_compiler_version:        # [linux]
```

Comment:
Now that things seem working. Can we try to remove this? I applied a GCC patch to Fonda forge a few days back

Reply:
I will do this once the linux builds pass again with GCC 9. I have introduced some kind of issue that I want to figure out before making a change there again.